assessment.md

```diff
@@ -94,34 +94,34 @@
   * :smiley: = Refactored code contains easy to understand functions and documentation.
   * :sunglasses: = Refactored code is clean, efficient, well documented, and maintainable.
 * Debugging Python programs using logging and other methods
-  * :disappointed: = No fault-localization and no debugging.
-  * :confused: = Identifies some of the bugs.
+  * :disappointed: = No fault-localization and no debugging
+  * :confused: = Identifies some of the bugs
   * :neutral_face: = Identifies and fixes some of the bugs using logging and other
-  methods.
-  * :smiley: = Identifies and fixes most of the bugs using logging and other methods.
+  methods
+  * :smiley: = Identifies and fixes most of the bugs using logging and other methods
   * :sunglasses: = Identifies and fixes most of the bugs using logging and other
-  methods, and documents them.
+  methods, and documents them
```

Comment:
It is unclear here. Is student required to document the bugs?

Reply:
Documentation should refer to the method, not the bug. However, I agree that it is unclear.